app/jinja_filters.py

```diff
@@ -100,6 +103,28 @@ def format_conditional_date(date1=None, date2=None):
     return format_date(date)
 
 
+@blueprint.app_template_filter()
+def calculate_years_difference(date_1, date_2):
+
+    from_date = convert_to_datetime(date_1)
+
+    if date_2 == 'now':
+
+        to_date = datetime.now()
+
+    else:
+
+        to_date = convert_to_datetime(date_2)
+
+    difference = relativedelta.relativedelta(to_date, from_date)
+    year_string = 'year' if difference.years == 1 else 'years'
```

Comment:
year_string = '{} year' if difference.years == 1 else '{} years'
Would make the following more efficient